operator/pkg/gateway-api/gatewayclass_reconcile.go

```diff
@@ -46,6 +46,29 @@ func (r *gatewayClassReconciler) Reconcile(ctx context.Context, req ctrl.Request
 	// Hence, just set gateway class Accepted condition to true blindly.
 	setGatewayClassAccepted(gwc, true)
 
+	// List of features supported by Cilium.
+	// The same is used in GHA CI .github/workflows/conformance-gateway-api.yaml
```

Comment:
the .github seems weird here?

Reply:
ah it's the full path from root directory to gha workflow file

Reply:
:facepalm: